pkgs/by-name/re/recutils/package.nix

```diff
@@ -4,6 +4,16 @@
 , bc
 , check
 , curl
+
+, withEncryption ? true
+, libgcrypt
+, libgpg-error
+
+, withUuid ? true
+, libuuid
+
+, withBashBuiltins ? true
```

Comment:
Because Bash builtins are uncommonly used, it may be better to default to `false` here.

Reply:
I think it should be true by default. It will hurt no one.